### PR TITLE
Add colReorder to pricing- and sample requirements pages

### PR DESCRIPTION
**Pull request by sofiahag**

### CODE REVIEW

**Comment by aanil** [comment]
Is this on stage?

**Comment by sofiahag** [comment]
It is now!

File: run_dir/static/js/pricing_main.js [hunk 1]
```diff
@@ -677,7 +677,8 @@ app.component('v-products-table', {
             this.dataTable = $('#pricing_products_table').DataTable({
                 "paging":false,
                 "info":false,
-                "order": [[ 1, "asc" ]]
+                "order": [[ 1, "asc" ]],
+                colReorder: true
```
Comment (aanil) [inline run_dir/static/js/pricing_main.js:681]: Breaks the addition of new products in Agreements and Cost calculator.

Comment (sofiahag) [reply run_dir/static/js/pricing_main.js:681]: I wasn't aware we could add new products, I don't think I have the right permissions 🤔 I get a server error. 
OK, I will have to look into it then 🤷🏻‍♀️

Comment (sofiahag) [reply run_dir/static/js/pricing_main.js]: We decided to not include the colReorder for pricing, it would require more work than it's worth 🤷🏻‍♀️

**Review by aanil — APPROVED** [review]
👍

**Pull request merged**